src/app/features/terms-of-use/terms-of-use.component.html

```diff
@@ -0,0 +1,54 @@
+<app-page>
+  <ng-container content>
+    <div class="bg-slate-50">
+      <div class="container mx-auto p-16 pt-44">
+        <div>
+          <p class="text-gray-500/90">
+            الرئيسية
+            <span class="px-2">
+              <i
+                class="fa-solid fa-chevron-left fa-sm"
+                style="color: #79808c"
```

Comment:
style is not good it can be removed
because the parent has the same color

> If You want to use deferent color you can use classes
`<i class="fa-solid .... text-gray-500"></i>`